src/components/ExtensibilitySpectrum/index.jsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import styles from "./index.module.css";
+import classNames from "classnames";
+
+const appConfigSvg = (
+    <switch>
+        <g>
+            <path fill="#fff" d="M68.4 89.2h151v110.2h-151z" />
+            <path
+                className={styles.svgFillBlue}
+                d="M219.4 202.7h-151c-1.8 0-3.3-1.5-3.3-3.3V89.2c0-1.8 1.5-3.3 3.3-3.3h151c1.8 0 3.3 1.5 3.3 3.3v110.2c0 1.8-1.5 3.3-3.3 3.3zm-147.7-6.6h144.4V92.5H71.7v103.6z"
+            />
+            <path
+                className={styles.svgFillBlue}
+                d="M218.4 112.4h-150c-1.8 0-3.3-1.5-3.3-3.3s1.5-3.3 3.3-3.3h150c1.8 0 3.3 1.5 3.3 3.3s-1.5 3.3-3.3 3.3z"
+            />
+            <g>
+                <path
+                    className={styles.svgFillBlue}
+                    d="M191.5 111.1c-1.8 0-3.3-1.5-3.3-3.3V89.2c0-1.8 1.5-3.3 3.3-3.3s3.3 1.5 3.3 3.3v18.6c0 1.8-1.5 3.3-3.3 3.3z"
+                />
+            </g>
+            <g>
+                <path
+                    className={styles.svgFillBlue}
+                    d="M141.8535 153.36022l5.42124-3.76464 19.6788 28.3383-5.42124 3.76464z"
+                />
+                <path
+                    className={styles.svgFillBlue}
+                    d="M129.4 170.3l-3.9-46.2 42 19.7z"
+                />
+            </g>
+        </g>
+    </switch>
+);
+
+const workflowSvg = (
+    <path
+        className={styles.svgFillBlue}
+        d="M15 7H9V1h6v6zM7 17H1v6h6v-6zm8 0H9v6h6v-6zm8 0h-6v6h6v-6zm-10-6V8h-2v3H3v5h2v-3h6v3h2v-3h6v3h2v-5h-8z"
+    />
+);
+
+const customCodeSvg = (
+    <switch>
+        <g>
```

Comment:
nit: there's an extra level of `g` here that can be removed i believe. Also `switch` shouldn't be needed but double check.

Reply:
Removed.